src/Integration.Vsix/SonarLintTagger/SinkManager.cs

```diff
@@ -45,17 +53,17 @@ public void Dispose()
             taggerProvider.RemoveSinkManager(this);
         }
 
-        internal void AddFactory(SnapshotFactory factory)
+        public void AddFactory(SnapshotFactory factory)
```

Comment:
The class is already _internal_ so I've changed the methods to _public_ to make it clearer that they are really part of the public API of the class.